@theme/components/Amendments.tsx

```diff
@@ -0,0 +1,391 @@
+import * as React from 'react';
+import { Link } from '@redocly/theme/components/Link/Link';
+import { useThemeHooks } from '@redocly/theme/core/hooks';
+
+type Amendment = {
+  name: string;
+  rippled_version: string;
+  tx_hash?: string;
+  consensus?: string;
+  date?: string;
+  id: string;
+  eta?: string;
+};
+
+type AmendmentsResponse = {
+  amendments: Amendment[];
+};
+
+type AmendmentsCachePayload = {
+  timestamp: number;
+  amendments: Amendment[];
+}
+
+// API data caching
+const amendmentsEndpoint = 'https://vhs.prod.ripplex.io/v1/network/amendments/vote/main/';
+const amendmentsCacheKey = 'xrpl.amendments.mainnet.cache';
+const amendmentsTTL = 12 * 60 * 60 * 1000; // 12 hours in milliseconds
```

Comment:
Where did 12 hours come from? Seems like overkill to me. Amendment statuses don't update _that_ often, but when they do, it's on a flag ledger which are approximately 15 minutes apart.

Seems to me like it would not cause excessive load to check amendments' status every 15 minutes instead of 12 hours, which _would_ probably give people a more gratifyingly "close to live" view of Mainnet's status.

Reply:
At the time I just picked an arbitrary number, but your suggestion makes sense.